pkg/ui/src/views/cluster/containers/nodesOverview/index.tsx

```diff
@@ -133,18 +137,30 @@ export class NodeList extends React.Component<LiveNodeListProps> {
       render: (_text, record) => {
         if (!!record.nodeId) {
           return (
-            <React.Fragment>
-              <Link className="nodes-table__link" to={`/node/${record.nodeId}`}>
-                <Text textType={TextTypes.BodyStrong}>{`N${record.nodeId} `}</Text>
-                <Text>{record.region}</Text>
-              </Link>
-            </React.Fragment>);
+            <Link className="nodes-table__link" to={`/node/${record.nodeId}`}>
+              <Text textType={TextTypes.BodyStrong}>{`N${record.nodeId} `}</Text>
+              <Text>{record.nodeName}</Text>
+            </Link>
+          );
         } else {
           // Top level grouping item does not have nodeId
           return (
-            <React.Fragment>
-              <Text>{record.region}</Text>
-            </React.Fragment>);
+            <Text>
+              <Tooltip
```

Comment:
Would be nice to extract the tooltip to a pure presentation component

Reply:
Sorry if I wrong, but Tooltip component is already defined as pure component.
In this case we provide only props (`title` and `children`) as inputs. Both of these props has
to be defined outside of tooltip component.
To make it more readable, I would extract entire render component.

Reply:
@dhartunian , please check if new changes somehow fit to yours proposed changes 🙄